scripts/cibuild

```diff
@@ -26,6 +26,12 @@ if [[ "${BASH_SOURCE[0]}" == "$0" ]]; then
         REACT_APP_GIT_COMMIT="${GIT_COMMIT}" \
             ./scripts/update
 
+        # Build static asset bundle
+        docker-compose \
+            -f docker-compose.yml \
+            run --rm --no-deps --entrypoint "bash -c" taui \
+            "yarn install && yarn build"
```

Comment:
Is this necessary if it's also happening in `scripts/update`?

Reply:
Ah, nope! Will take this out and leave the `yarn install` and `build` in `scripts/update` to cover asset bundling for both local and deployed setup.

Reply:
This worked for me! This is back to being out of `scripts/cibuild` 🎉